### PR TITLE
Add myself lucaslattion to the list of students

### LINKED ISSUE #57: Add myself (@lucaslattion ) to the list of students
Add myself (@lucaslattion ) to the list of students

### PR DESCRIPTION
**Pull request by lucaslattion**
@ludelafo : Please add myself (@lucaslattion ) to the list of students
Fixes #57

### CODE REVIEW

**Comment by lucaslattion** [comment]
Add myself (@lucaslattion ) to the list of students

**Comment by lucaslattion** [comment]
@teaching-staff : Please add myself (@lucaslattion ) to the list of students

**Comment by lucaslattion** [comment]
Fixes #56

File: profile/README.md [hunk 1]
```diff
@@ -38,5 +38,6 @@ First name Last name [@GitHub username](https://github.com/USERNAME)
 - Alan Sottile [@AlanSottile](https://github.com/AlanSottile)
 - Jeremiah Steiner [@BetterBlood](https://github.com/BetterBlood)
 - Jonas Troeltsch [@JonasTroeltsch](https://github.com/JonasTroeltsch)
+- Lucas Lattion [@lucaslattion](https://github.com/lucaslattion)
```
Comment (ludelafo) [inline profile/README.md]: Please add your name in alphabetical order by last name :)

**Review by ludelafo — CHANGES_REQUESTED** [review]
Could you sign your commits as well? You can follow the documentation here: https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/03-git-github-and-markdown/COURSE_MATERIAL.md#make-changes-if-needed

**Review by hadrylouis — APPROVED** [review]

**Pull request merged**